site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,554 @@
+- [Hidden, yet searchable and accessible DOM](https://github.com/whatwg/html/issues/6040).
+- [Shared element transitions](https://github.com/WICG/shared-element-transitions).
+- [Custom layout](https://www.w3.org/TR/css-layout-api-1/).
+- Off-main-thread compositing; decoupling threading and compositing.
+
+## Key projects that make up RenderingNG
+
+Below is a list of the key projects within RenderingNG. Subsequent blog posts will deep-dive into each of them.
+
+### CompositeAfterPaint
+
+Disentangles compositing from style, layout and paint,
+allowing much-improved reliability and predictable performance, increased throughput,
+and using less memory without sacrificing performance. It began in 2014 and will finish this year.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2015</td>
+      <td>Ship display lists.</td>
+    </tr>
+    <tr>
+      <td>2017</td>
+      <td>Ship new invalidation.</td>
+    </tr>
+    <tr>
+      <td>2018</td>
+      <td>Ship property trees part 1.</td>
+    </tr>
+    <tr>
+      <td>2019</td>
+      <td>Ship property trees part 2.</td>
+    </tr>
+    <tr>
+      <td>2021</td>
+      <td>Ship SVG.</td>
+    </tr>
+    <tr>
+  </tbody>
+</table>
+
+### LayoutNG
+
+A ground-up rewrite of all layout algorithms,
+for greatly improved reliability and more predictable performance. It began in 2016
+and is planned to finish this year.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2019</td>
+      <td>Ship block flow.</td>
+    </tr>
+    <tr>
+      <td>2020</td>
+      <td>Ship flex, editing.</td>
+    </tr>
+    <tr>
+      <td>2021</td>
+      <td>Ship everything else.</td>
+    </tr>
+    <tr>
+  </tbody>
+</table>
+
+### BlinkNG
+
+A systematic cleanup and refactoring of the Blink rendering engine into cleanly separated pipeline phases.
+This allows for better caching, higher reliability,
+and re-entrant / delayed-rendering features such as content-visibility and container queries. It began in 2014, and incremental improvement and has been ongoing since. It will complete in 2021.
+
+### GPU Acceleration everywhere
+
+A long-term effort to roll out GPU rasterization, draw and animation on all platforms, all of the time.
+GPU acceleration provides an enormous speedup for most content,
+because every pixel can be processed in parallel.
+It is also an effective method for improving performance on low-end devices, which tend to still have a GPU.
+It began in 2014 and completed in 2020.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2014</td>
+      <td>Canvas support. Shipped on opt-in content on Android.</td>
+    </tr>
+    <tr>
+      <td>2016</td>
+      <td>Ship on Mac.</td>
+    </tr>
+    <tr>
+      <td>2017</td>
+      <td>GPU is used on &gt;60% of Android page views.</td>
```

Comment:
```suggestion
      <td>GPU is used on over 60% of Android page views.</td>
```

Reply:
Words as words again, MacOS screen reader would read this as just "used on 60% of Android page views".